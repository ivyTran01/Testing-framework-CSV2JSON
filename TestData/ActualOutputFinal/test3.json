[{"name":"Ivy","age":22,"ice_cream_lover":" true \"inside\" end "},{"name":"Tanzil","age":25,"ice_cream_lover":false},{"name":"Danny","age":21,"ice_cream_lover":true}]