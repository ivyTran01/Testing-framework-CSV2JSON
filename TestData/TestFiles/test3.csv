name,age,ice_cream_lover
Ivy,22," true ""inside"" end "
Tanzil,25,false
Danny,21,true
