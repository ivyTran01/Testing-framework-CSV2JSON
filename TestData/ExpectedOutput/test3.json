[
 {
  "field1":"Ivy",
  "field2":22,
  "field3":"true",
  "field4":5.2
 },
 {
  "field1":"Tanzil",
  "field2":25,
  "field3":" false; ",
  "field4":5.9
 },
 {
  "field1":"Danny",
  "field2":21,
  "field3":"true",
  "field4":5.11
 },
 {
  "field1":"Abdullah",
  "field2":23,
  "field3":"true",
  "field4":5.1
 },
 {
  "field1":"Jocelyn",
  "field2":22,
  "field3":"true",
  "field4":5.2
 },
 {
  "field1":"Christ",
  "field2":23,
  "field3":"false",
  "field4":5.1
 }
]